### PR TITLE
chore: remove repetitive words

### PR DESCRIPTION
**Pull request by avoidaway**
**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

```markdown changelog

```

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by thaJeztah** [comment]
Thanks! Could you use your real name (not github handle) for the sign-off?

```
Signed-off-by: avoidaway <cmoman@126.com> 
```

See https://github.com/moby/moby/blob/master/CONTRIBUTING.md#sign-your-work

> Use your real name (sorry, no pseudonyms or anonymous contributions.)

File: contrib/download-frozen-image-v2.sh [hunk 1]
```diff
@@ -11,7 +11,7 @@ set -eo pipefail
 # scope, but may not accept contributions beyond that.
 #
 # For users who have a similar need but require more flexibility/functionality,
-# refer the the discussion on GitHub, which mentions various alternatives that
+# refer the discussion on GitHub, which mentions various alternatives that
```
Comment (thaJeztah) [inline contrib/download-frozen-image-v2.sh:14]: Looks like this one is missing a word; should probably be `refer to the discussion ....`

Comment (avoidaway) [reply contrib/download-frozen-image-v2.sh]: Thanks!  Modified

**Comment by avoidaway** [comment]
friendly ping~

**Review by thaJeztah — APPROVED** [review]
LGTM

**Pull request merged**